book-service/src/main/java/com/jpop/bookservice/controller/BookController.java

```diff
@@ -0,0 +1,52 @@
+package com.jpop.bookservice.controller;
+
+import com.jpop.bookservice.model.Book;
+import com.jpop.bookservice.service.BookService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.List;
+import java.util.Optional;
+
+@RestController
+@RequestMapping("/")
+public class BookController {
+
+    @Autowired
+    BookService bookService;
+
+    @GetMapping("books")
+    List<Book> getAllBooks(){
+        return bookService.findAll();
+    }
+
+    @GetMapping("books/{id}")
```

Comment:
Try to follow versioning.
https://dzone.com/articles/versioning-rest-api-with-spring-boot-and-swagger

Reply:
Resolved